swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Clothing(Item):
+
+    def __init__(self, id=None, condition=0, fabric="Unknown"):
+        super().__init__(id=id, condition=condition)
+        self.fabric = fabric
+
+    def __str__(self):
+        return f"An object of type Clothing with id {self.id}. It is made from {self.fabric} fabric."
```

Comment:
Notice that the first part of your return string (f"An object of type Clothing with id {self.id}.") looks very similar to your parent class's __str__ method! It is possible for us to add on to what is already being returned from the super class by calling its dunder string!
```suggestion
        return f"{super().__str__()} It is made from {self.fabric} fabric."
```